lib/ffmpeg/vaapi/transcoder.py

```diff
@@ -36,7 +36,7 @@ class TranscoderTest(BaseTranscoderTest):
         hw = (platform.get_caps("decode", "vc1"), have_ffmpeg_decoder("vc1"), "vc1"),
       ),
       "av1" : dict(
-        hw = (platform.get_caps("decode", "av1_8"), have_ffmpeg_decoder("av1"), "av1"),
+        hw = (platform.get_caps("decode", "av1_8"), have_ffmpeg_decoder("av1_qsv"), "av1_qsv"),
```

Comment:
@uartie ,can I modify it directly?

Reply:
Right.  Please don't use qsv plugins in vaapi context.  We should not assume qsv is available when running vaapi cases.

Reply:
emmm,but when I run vaapi, iopts will get "ffmpeg - v verbose - c: v av1 ...", It is still not supported during verification.

Reply:
Because you need to setup the vaapi hwaccel parameters (i.e. see the TODO in my review)

Reply:
OOP principles are necessary.